### PR TITLE
statsd work in progress

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: shoebox/app/com/keepit/common/performance/package.scala [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
+    sw.stop()
     sw.logTime()
     res
   }
+  def timeWithStatsd[A](tag: String, statsdTag: String)(f: => A): A = {
+    val sw = new Stopwatch(tag)
+    val res = f
+    sw.stop()
+    sw.logTime()
+    Statsd.timing(statsdTag, sw.elapsedTime / 1000)
```
Comment (yingjieMiao) [inline shoebox/app/com/keepit/common/performance/package.scala:32]: should we divide by10^6 ?

Comment (eishay) [inline shoebox/app/com/keepit/common/performance/package.scala:32]: Mili is good enough

Comment (yingjieMiao) [inline shoebox/app/com/keepit/common/performance/package.scala:32]: nano = 10^(-9)
micro = 10^(-6)
milli = 10^(-3)
We are converting from nano to milli, right?

Comment (eishay) [inline shoebox/app/com/keepit/common/performance/package.scala:32]: yes, you're right :-)

Comment (gmethvin) [inline shoebox/app/com/keepit/common/performance/package.scala:32]: oh, right

Comment (gmethvin) [inline shoebox/app/com/keepit/common/performance/package.scala:32]: fixed

**Pull request merged**